### DISCUSSION TITLE
Busy

### DISCUSSION THREAD

**Discussion by rodit** [rodit/SnapMod#192] [Announcements]
Just so everyone knows, I will be very busy over the next couple of months so SnapMod development will slow down significantly.

Thanks,
rodit

**Comment by Magna-Graecia** [comment]
Me sad :cry:

**Comment by snaphappy1** [comment]
Snapmod has exceeded all my expectations and as far as I'm concerned this mod was complete several versions ago, everything since then has been icing on the cake. Thank you so much for your work. Enjoy the time off and good luck with your future endeavors!

**Reply by Magna-Graecia** [reply]
Bro I thought you disappeared haha, but I 100% agree. Can you try sending a video snap with 'Disable Snap Splitting' and 'Force Source Encoding' both disabled? It needs to be at least 20 seconds long, but preferably around a minute if you have one around that long. My split snaps are freezing beginning with the 2nd snap in the sequence. I know you have the same setup as me, so I figured I would ask and see if you have the same issue.

**Reply by snaphappy1** [reply]
Tried with the latest sc and snapmod, unfortunately I could not recreate the problem. Both the options you mentioned were disabled and the videos were a minute long which sc split into 6. They sent and were viewed without issues, no freezing or hiccups. If you haven't already, I'd try re-installing sc and snapmod, maybe rebooting in between to make sure the cache is cleared...

**Comment by drickles** [comment]
Bruh. The mod works perfectly and flawlessly. It's undetectable and has every feature I could ever dream of. On top of that it supports the absolutely latest version on the app. You prolly could take a 2 year break from the project and it would still work flawlessly without a glitch. You good bro. Thanks for the hard work. Take as much time off as you want. Due to the excellent work you did your project got a long lifespan.

**Reply by LiqwidSw** [reply]
100% agree with this, In it's current state SnapMod does what I need it to do and hopefully it will be a while before the powers that be catch up to breaking it. @rodit was putting out updates at a breakneck speed for a while there, take time for yourself.

**Comment by ghost** [comment]
i'll provide some version updates ig <3

**Reply by NMBlanco** [reply]
Please :(

**Comment by RevealedSoulEven** [comment]
It's been a while. I open github everyday to check whether there is an update or not🥺
Many features are broke in newer snapchat which only you could fix🥺
I've tried and resolved most of the features but still lack of knowledge in RE and Debugging I can't understand how the app is working and which function is called at what instance so that I could continue the project.
Please come back✨

**Comment by rodit** [comment]
I would like to continue development soon. I will likely delete/archive every open issue and let people reopen issues for the latest version once updated.

**Reply by authorisation** [reply]
Thank you 🙏

**Reply by RevealedSoulEven** [reply]
Very Very eager to see you back soon.
